cl/citations/annotate_citations.py

```diff
@@ -78,16 +79,15 @@ def create_cited_html(
             source_text=opinion.source_text,
             unbalanced_tags="skip",  # Don't risk overwriting existing tags
         )
-    else:  # Else, make sure to wrap the new text in <pre> HTML tags...
+    else:  # Else, present `source_text` wrapped in <pre> HTML tags...
         new_html = annotate_citations(
             plain_text=opinion.cleaned_text,
             annotations=[
                 [a[0], f"</pre>{a[1]}", f'{a[2]}<pre class="inline">']
                 for a in generate_annotations(citation_resolutions)
             ],
-            source_text=opinion.source_text,
+            source_text=f'<pre class="inline">{html.escape(opinion.source_text)}</pre>',
```

Comment:
This looks fine to me, but I'm confused how we can just wrap stuff in a `pre`, and not face bad consequences. Can you help me, since I haven't looked at this part of the code in forever? I'd have Flooie look, but it's his last day before a vacation...

Reply:
The easy answer is basically, what’s sauce for the goose is sauce for the gander: above, in the `source_is_html` case, `source_text` has the same significant difference (HTML with tags) from `plain_text` that the non-HTML case now has.

Within Eyecite, the `SpanUpdater` does some complicated stuff to map the citation indices generated by `get_citations` which are relative to `plain_text` to indices on the HTML `source_text`. It’s a process that sounds fraught, but evidently works well enough.

Reply:
Cool, thanks. I also didn't see you removed line 90, so now it makes more sense.